src/services/TechRecordsService.ts

```diff
@@ -373,6 +418,55 @@ class TechRecordsService {
         }
     return new HTTPResponse(200, this.formatTechRecordItemForResponse(updatedTechRecord.Attributes as ITechRecordWrapper));
   }
+
+  public addProvisionalTechRecord(techRecord: ITechRecordWrapper, msUserDetails: IMsUserDetails) {
+    return this.addNewProvisionalRecord(techRecord, msUserDetails)
+    .then((data: ITechRecordWrapper) => {
+      return this.techRecordsDAO.updateSingle(data)
+        .then((updatedData: any) => {
```

Comment:
Could we use a type instead of any maybe?
What do you think?

Reply:
added the type